common/src/main/res/values/strings.xml

```diff
@@ -1147,4 +1147,29 @@
     <string name="state_arming">Arming</string>
     <string name="state_disarming">Disarming</string>
     <string name="state_pending">Pending</string>
+    <string name="state_mowing">Mowing</string>
+    <string name="state_above_horizon">Above horizon</string>
+    <string name="state_active">Active</string>
+    <string name="state_below_horizon">Below horizon</string>
+    <string name="state_buffering">Buffering</string>
+    <string name="state_cloudy">Cloudy</string>
+    <string name="state_exceptional">Exceptional</string>
+    <string name="state_clear_night">Clear night</string>
```

Comment:
This should include a comma

https://github.com/home-assistant/core/blob/57144a6064db0ff97cdc536580a329381bc3b1d3/homeassistant/components/weather/strings.json#L7

Reply:
Actually we don't need to duplicate the string as it isn't using reflection, so this can be removed?